pkg/pb/overlay.proto

```diff
@@ -85,6 +86,16 @@ message QueryResponse {
 message PingRequest {};
 message PingResponse {};
 
+// TODO: add fields that validate who is requesting the info
+message InfoRequest {}
+
+message InfoResponse {
+    bytes id = 1 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
+    node.NodeType type = 2;
+    node.NodeMetadata metadata = 3;
```

Comment:
Is there a better name for this? `NodeOperator` maybe?

Reply:
Same here. The `NodeMetadata` is an already existing type. It is not defined in this PR.

Reply:
I'm aware that these are already defined, it's more of we can improve the naming more easily right now, instead of later when it's integrated into elsewhere.

Reply:
Also completely fine to handle it later.

Reply:
OK. I will define a new `NodeOperator` here and remove `NodeMetadata` in #1341.

Reply:
Done.